x-pack/plugins/canvas/public/lib/sync_filter_expression.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { fromExpression } from '@kbn/interpreter/common';
+// @ts-ignore
+import { set, del } from 'object-path-immutable';
+import { get } from 'lodash';
+
+export function syncFilterExpression(
+  config: Record<string, any>,
```

Comment:
Is this truly `any`?  Or is it `string | number | null`?

Reply:
Yeah, pretty much. Nothing is used of enforced in this file, it just uses whatever values existing in the object. It can also be more than just `string | number | null`, at a minimum it's `string | number | null | Array<string | number | null>`, no idea what other kinds of values are valid there.